UnitTests/Sources/EmojiProviderTests.swift

```diff
@@ -19,45 +19,53 @@ import XCTest
 @testable import ElementX
 
 final class EmojiProviderTests: XCTestCase {
-    var sut: EmojiProvider!
-    private var emojiLoaderMock: EmojiLoaderMock!
-    
-    @MainActor override func setUp() {
-        emojiLoaderMock = EmojiLoaderMock()
-        sut = EmojiProvider(loader: emojiLoaderMock)
-    }
-    
-    func test_whenEmojisLoaded_categoriesAreLoadedFromLoader() async throws {
+    func testWhenEmojisLoadedCategoriesAreLoadedFromLoader() async throws {
         let item = EmojiItem(label: "test", unicode: "test", keywords: ["1", "2"], shortcodes: ["1", "2"], skins: ["🙂"])
         let category = EmojiCategory(id: "test", emojis: [item])
+        
+        let emojiLoaderMock = EmojiLoaderMock()
         emojiLoaderMock.categories = [category]
-        let categories = await sut.getCategories()
+        
+        let emojiProvider = await EmojiProvider(loader: emojiLoaderMock)
+        
+        let categories = await emojiProvider.getCategories()
```

Comment:
Whilst we're here, can we do?
```suggestion
        let categories = await emojiProvider.categories()
```